.gitignore

```diff
@@ -63,27 +63,29 @@ WordPress/Derived Sources/
 WordPress/Images.xcassets/AppIcon-Internal.appiconset
 
 # Bundler
-/vendor/bundle/
-.bundle/
+.bundle
 
 # Dependencies
-/vendor/
+vendor
 
 # Fastlane
-Scripts/fastlane/Preview.html
-Scripts/fastlane/report.xml
-Scripts/fastlane/README.md
-Scripts/Preview.html
-Scripts/fastlane/test_output/
-Scripts/fastlane/google_cloud_keys.json
-Scripts/fastlane/promo_screenshots
-/Scripts/fastlane/metadata/review_information
-/Scripts/default.profraw
+fastlane/Preview.html
+fastlane/report.xml
+fastlane/README.md
+Preview.html
+fastlane/test_output/
+fastlane/google_cloud_keys.json
+fastlane/promo_screenshots
```

Comment:
Those `fastlane/promo_screenshots` were added and commited as part of this PR, _before_ you updated the path to `.gitignore`. Which means that they are now erroneously into the repo.

Reply:
Shoot, yep. Fixed in d76a8e6ae4450360a8842f3dd0c1e0e85c1ee591